Applications/DataExplorer/VtkVis/VtkCompositeNodeSelectionFilter.h

```diff
@@ -28,7 +28,7 @@ class VtkCompositeNodeSelectionFilter : public VtkCompositeFilter
 
     void init() override;
 
-    /// Sets the point indeces to be highlighted
+    /// Sets the point indices to be highlighted
```

Comment:
Actually there is the same spelling error in `const std::vector<unsigned> &point_indeces` in the line below, but nevermind.